packages/player/src/components/video.tsx

```diff
@@ -28,100 +44,26 @@ export const Video: React.FC<VideoProps> = ({
   id,
   className,
 }) => {
-  const videoElemRef = React.createRef<HTMLVideoElement>()
+  //if you type this to an HTMLVideoElement it becomes read only
+  //and below we need to manually assign it so we can dispatch
+  //to the videoService and update our state machine context
+  const videoElemRef = React.useRef<any>(null)
   const {videoService} = React.useContext(VideoContext)
-  function handleLoadStart(event: SyntheticEvent) {
-    // console.log(event)
-  }
-
-  function handleWaiting(event: SyntheticEvent) {
-    //console.log(event)
-  }
-
-  function handleCanPlayThrough(event: SyntheticEvent) {
-    //console.log(event)
-  }
-
-  function handlePlaying(event: SyntheticEvent) {
-    //console.log(event)
-  }
-
-  function handleEnded(event: SyntheticEvent) {
-    //console.log(event)
-  }
-
-  function handleSeeking(event: SyntheticEvent) {
-    //console.log(event)
-  }
-
-  function handleSeeked(event: SyntheticEvent) {
-    //console.log(event)
-  }
-
-  function handlePlay(event: SyntheticEvent) {
-    //console.log(event)
-  }
-
-  function handlePause(event: SyntheticEvent) {
-    //console.log(event)
-  }
-
-  function handleProgress(event: SyntheticEvent) {
-    //console.log(event)
-  }
-
-  function handleDurationChange(event: SyntheticEvent) {
-    //console.log(event)
-  }
-
-  function handleError(event: SyntheticEvent) {
-    //console.log(event)
-  }
-
-  function handleSuspend(event: SyntheticEvent) {
-    //console.log(event)
-  }
-
-  function handleAbort(event: SyntheticEvent) {
-    //console.log(event)
-  }
-
-  function handleEmptied(event: SyntheticEvent) {
-    //console.log(event)
-  }
-
-  function handleStalled(event: SyntheticEvent) {
-    //console.log(event)
-  }
-
-  function handleLoadedMetaData(event: SyntheticEvent) {
-    //console.log(event)
-  }
-
-  function handleLoadedData(event: SyntheticEvent) {
-    //console.log(event)
-  }
-
-  function handleRateChange(event: SyntheticEvent) {
-    //console.log(event)
-  }
-
-  function handleVolumeChange(event: SyntheticEvent) {
-    //console.log(event)
-  }
 
   return (
     <video
       className={`cueplayer-react-video ${className}`}
       id={id}
       crossOrigin={crossOrigin}
       ref={(c: HTMLVideoElement) => {
```

Comment:
using the callback approach here because I want to send the ref object to xstate